src/Microsoft.AspNetCore.Mvc.Core/ControllerBase.cs

```diff
@@ -1306,15 +1306,15 @@ public virtual ChallengeResult Challenge(AuthenticationProperties properties)
             => new ChallengeResult(authenticationSchemes, properties);
 
         /// <summary>
-        /// Creates a <see cref="ForbidResult"/>.
+        /// Creates a <see cref="ForbidResult"/> (HTTP 403).
```

Comment:
Maybe say `(HTTP 403 by default)`? I want to make sure we don't throw the baby out with the bathwater here.

I'd be fine if, for example, in the `<remarks>` section it mentioned that some authentication schemes, such as cookies, will typically convert the HTTP 403 to an HTTP redirect to show the user a login page.